android/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ buildscript {
 }
 
 plugins {
+  `project-properties`
```

Comment:
this change @dubdabasoduba

Reply:
This was to allow `property("MAPBOX_SDK_TOKEN").toString()`

Reply:
Please test if `property("MAPBOX_SDK_TOKEN").toString()` can work without it

Reply:
just checked, thrown an exception 
Exception is:
groovy.lang.MissingPropertyException: Could not get unknown property 'MAPBOX_SDK_TOKEN' for root project

so this must be there.